flake/dev/all-maintainers.nix

```
@@ -0,0 +1,56 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      apps.all-maintainers = {
+        program = pkgs.writeShellApplication {
+          name = "update-all-maintainers";
+          runtimeInputs = [ pkgs.gitMinimal ];
+          text = ''
+            root="$(git rev-parse --show-toplevel)"
+            cp --force \
+              ${config.packages.all-maintainers} \
+              "$root/stylix/generated/all-maintainers.nix"
+          '';
+        };
+        meta.description = "update all-maintainers.nix";
+      };
+
+      packages.all-maintainers =
+        pkgs.runCommand "all-maintainers"
+          {
+            passAsFile = [ "maintainers" ];
+            maintainers =
+              ''
+                # DO NOT EDIT THIS GENERATED FILE.
+                #
+                # This file lists Stylix module maintainers for GitHub review
+                # requests, per NixOS RFC 39.
+                #
+                # To generate this file, run:
+                #
+                #       nix run .#all-maintainers
+              ''
+              + lib.pipe ../../stylix/meta.nix [
+                (p: pkgs.callPackage p { })
+                builtins.attrValues
+                (builtins.concatMap (target: target.maintainers or [ ]))
+                (map (maintainer: lib.nameValuePair maintainer.github maintainer))
+                builtins.listToAttrs
+                (lib.generators.toPretty { })
+              ];
+            inherit (config.treefmt) projectRootFile;
+            nativeBuildInputs = [ config.formatter ];
+          }
+          ''
+            touch "$projectRootFile"
+
+            cp "$maintainersPath" result.nix
+
+            treefmt --no-cache result.nix
+
+            install -m 644 -T result.nix "$out"
```

Now that there is no longer multiple levels of indentation, I wonder if this reads better without blank lines?


```suggestion
            touch "$projectRootFile"
            cp "$maintainersPath" result.nix
            treefmt --no-cache result.nix
            install -m 644 -T result.nix "$out"
```